src/changes/changes.xml

```diff
@@ -22,6 +22,7 @@
   <body>
 
   <release version="3.5" date="tba" description="tba">
+    <action issue="LANG-1175" type="fix" dev="wikier" dute-to="Benedikt Ritter">Remove Ant-based build</action>
```

Comment:
Copy and paste error. Should be `due-to`. I've corrected this for the other lines.